pkg/config/polling_manager.go

```diff
@@ -94,7 +94,10 @@ func (cm *PollingProjectConfigManager) SyncConfig(datafile []byte) {
 
 		if e != nil {
 			cmLogger.Error(fmt.Sprintf("request returned with http code=%d", code), e)
```

Comment:
I think we can tone this down to a `Warning`. @mjc1283 or @mikecdavis what do you guys think?

Reply:
Chatted with @mjc1283 offline and this is in line with what the other SDKs do.